### ISSUE TITLE
How to find tag with namespace?

### ISSUE THREAD

**Issue by fan-y-ouchi** [philss/floki#46]
```
iex(1)> "<ns:tag>xxx</ns:tag>" |> Floki.find("ns:tag")
Unknown token ':'. Ignoring.
[]
```

The jQuery can do as follows:
`$("ns\\:tag")`

**Comment by philss** [comment]
Hi @fan-y-ouchi! This feature was not present yet. But is now available in the version `0.8.0` of Floki. :smiley: 

Please update and check again (the same syntax of your example).

**Comment by fan-y-ouchi** [comment]
Thank you! :smile:

**Comment by gmile** [comment]
@philss will this not conflict with possible future implementation of `:first-child`, `:nth-child()` etc.? Per ** 6.1.1. Type selectors and namespaces** from [here](https://www.w3.org/TR/css3-selectors/#typenmsp), one must target namespaced tags by means of dash symbol `|`.

@philss what do you think?

**Comment by philss** [comment]
@gmile wow! I didn't know that. I think I misunderstood this namespace selector. Thank you for pointing out. This need to be fixed. I'm gonna reopen this.

**Comment by philss** [comment]
Closed again by #58.

**Comment by philss** [comment]
@fan-y-ouchi @gmile the new version was released in v0.10.0. Thanks!
